pages/community/community-projects.md

```diff
@@ -8,8 +8,8 @@ redirect_from: /testimonials/
 ---
 
 {{site.data.alerts.note}}
-Do you use preCICE? We are always happy to hear about interesting projects that use preCICE. <br>
-<a class="button primary" href="community-channels.html">Tell us your story!</a>
+Do you use preCICE? We are always happy to hear about interesting projects that use preCICE. To add your project here, you can directly open a PR to the website. Simply copy any of the <a class="button primary" href="https://github.com/precice/precice.github.io/tree/master/collections/_testimonials"> previous descriptions</a>, adjust with your data, and <a class="button primary" href="https://github.com/precice/precice.github.io/tree/master/images/testimonials">add a picture</a>.<br>
```

Comment:
Two suggestions:
- Many of the people we target here may not already know how to do this. I added a bit more details and a link to the GitHub documentation.
- Do we really need to change the line for the example? It feels disconnected.

```suggestion
Do you use preCICE? We are always happy to hear about interesting projects that use preCICE. To add your project here, you can directly [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to the website repository on GitHub (it is easy also if you don't yet have any GitHub experience!). Simply copy any of the <a class="button primary" href="https://github.com/precice/precice.github.io/tree/master/collections/_testimonials"> previous descriptions</a>, adjust with your data, and <a class="button primary" href="https://github.com/precice/precice.github.io/tree/master/images/testimonials">add a picture</a>.
```

Reply:
Agree. Have you checked / do you know whether the md formatting (linking) works here?

Reply:
I did not check the particular instance, but it does work elsewhere. If you commit my suggestion, we can also download the built website from the artifacts before we merge.